### PR TITLE
Merge Ansenn changes

### PR DESCRIPTION
**Pull request by Ansenn**
Changes : 
- Ansenn PPP
- Edit Clear Command
- Updated DG for Clear Command
- Added more JUnit Test

### CODE REVIEW

File: docs/team/ansenn.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - Wrote and edited javadoc for some methods
+  
+- **Review/mentoring contributions:**:
+  - PRs reviewed : [\#15](https://github.com/AY2223S2-CS2113-T11-2/tp/pull/15), [\#43](https://github.com/AY2223S2-CS2113-T11-2/tp/pull/43)
+
+- **Contributions beyond the project team**:
+  - Reported 11 bugs for PE-D
+
+- **Contributions to the Developer Guide (Extracts)**:
+  Logic object diagram:
```
Comment (bentohset) [inline docs/team/ansenn.md:51]: Perhaps this is supposed to be clear command sequence diagram?

**Review by bentohset — COMMENTED** [review]
Probably a typo

File: docs/team/ansenn.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- **Code contributed**: [RepoSense link](https://nus-cs2113-ay2223s2.github.io/tp-dashboard/?search=ansenn&breakdown=true)
+
+- **Contributions to UG**:
+  - Edit formatting of various commands in the UG
+  - Improved clarity of the UG in response to PE-D comments 
+  - Written Documentation for:
+    - List Command
+    - Help Command
+    - Clear Command
+  - Sequence diagram for Clear Command
```
Comment (TayJiunYuan) [inline docs/team/ansenn.md:31]: Perhaps sequence diagram should be under DG instead?

Comment (sriram-senthilkr) [reply docs/team/ansenn.md:31]: I believe it can be under the (Extracts) for his PPP?

File: docs/team/ansenn.md (hunk 1 above)
Comment (Ansenn) [reply docs/team/ansenn.md]: Yes oops

File: docs/team/ansenn.md (hunk 2 above)
Comment (Ansenn) [reply docs/team/ansenn.md]: Dear Jiun Yuan and Sriram,

I hope this comment finds you well. I just wanted to take a moment to thank you for taking the time to review my pull request. Your feedback and suggestions have been incredibly helpful in improving my code, and I truly appreciate your efforts.

Your expertise and attention to detail are invaluable, and I am grateful for the opportunity to work with you. Please let me know if there is anything else I can do to improve the code, or if you have any further suggestions or feedback.

Once again, thank you for your help and support. I look forward to continuing to work with you.

Best regards,
Ansenn

**Pull request merged**